### PR TITLE
Updated integration test docs in README.

### PR DESCRIPTION
**Pull request by jrbenny35**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -297,9 +297,12 @@ For a full reference of all the common commands that can be run inside the conta
 
 
 
-#### make integration_test
+#### make integration_test_legacy
 
-Run the integration test suite inside a containerized instance of Firefox. You must also be already running a `make up` dev instance in another shell to run the integration tests.
+Run the integration test suite for experimenter inside a containerized instance of Firefox. You must also be already running a `make up` dev instance in another shell to run the integration tests.
+#### make integration_test_nimbus
```
Comment (pdehaan) [inline README.md:303]: Nit:  can we add a blank line before the h4 header here?

File: README.md [hunk 2]
```diff
@@ -313,6 +316,15 @@ tox -c tests/integration/
...
 This should run the integration tests and watch them run in a Firefox instance you can watch and interact with.
 
+#### Integration Test options
+
+- `TOX_ENV`: [Tox](https://tox.readthedocs.io/en/latest/config.html#tox) commandline variables.
+- `PYTEST_ENV`: [Pytest](https://docs.pytest.org/en/6.2.x/usage.html#) commandline variables.
+
+An example using PYTEST_ARGS to run one test.
+```bash
+make integration_test_legacy PYTEST_ARGS="-k test_addon_rollout_experiment_e2e
```
Comment (pdehaan) [inline README.md:326]: nit: Add closing `"` at end of command?

File: README.md [hunk 3]
```diff
@@ -313,6 +316,15 @@ tox -c tests/integration/
 
 This should run the integration tests and watch them run in a Firefox instance you can watch and interact with.
 
+#### Integration Test options
+
+- `TOX_ENV`: [Tox](https://tox.readthedocs.io/en/latest/config.html#tox) commandline variables.
+- `PYTEST_ENV`: [Pytest](https://docs.pytest.org/en/6.2.x/usage.html#) commandline variables.
```
Comment (pdehaan) [inline README.md:322]: Q: Should these be `TOX_ENV` or `TOX_ARGS` (to match L157+L160 in the Makefile above)?
(same w/ `PYTEST_ENV` vs `PYTEST_ARGS`)

File: README.md [hunk 4]
```diff
@@ -313,6 +316,15 @@ tox -c tests/integration/
...
+
+An example using PYTEST_ARGS to run one test.
+```bash
+make integration_test_legacy PYTEST_ARGS="-k test_addon_rollout_experiment_e2e
+```
 ## Accessing Remote Settings locally
```
Comment (pdehaan) [inline README.md:328]: nit: add empty line between the closing code block and H2 header here.

File: README.md [hunk 5]
```diff
@@ -297,9 +297,12 @@ For a full reference of all the common commands that can be run inside the conta
 
 
 
-#### make integration_test
+#### make integration_test_legacy
```
Comment (pdehaan) [inline README.md:300]: :+1: good catch!

**Review by pdehaan — APPROVED** [review]
R+ with a couple nits and questions.

File: README.md (hunk 3 above)
Comment (jrbenny35) [reply README.md:322]: Ah, good catch!

**Pull request merged**